.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/code
+    docker:
+      - image: circleci/android:api-27-alpha
+    environment:
+      JVM_OPTS: -Xmx3200m
+      ANDROID_LICENSES: "/opt/android/sdk/licenses"
+    steps:
+      - run:
+          name: Accept Licenses
+          command: mkdir -p $ANDROID_LICENSES &&
+                   echo d56f5187479451eabf01fb78af6dfcb131a6481e > $ANDROID_LICENSES/android-sdk-license &&
```

Comment:
where did these keys come from? should they be public?

Reply:
Seems to be a large problem in the community. I found the solution in a few posts, but this is the source:
https://stackoverflow.com/questions/38210675/circleci-android-constraintlayout-doesnt-work/43539858

Reply:
I see.  Cool, thanks :)